src/pages/requests/RequestSummary.tsx

```diff
@@ -53,6 +64,80 @@ const RequestSummary: FunctionComponent<IRequestSummaryProps> = ({
   const [typingTimeout, setTypingTimeout] = useState<any>(null);
   const [prom, setProm] = useState<any>(Promise.resolve());
 
+  const [rowData, setRowData] = useState([
+    { cmoSampleName: "", cmoPatientId: "", investigatorSampleId: "" },
+    { cmoSampleName: "", cmoPatientId: "", investigatorSampleId: "" }
+  ]);
```

Comment:
are these meant to be here? I am a little unclear with what these are for.

Reply:
I removed.